platform/admin/core/admin-core-logviewer/src/main/webapp/js/filter.js

```diff
@@ -18,32 +18,41 @@ import includes from 'lodash/includes'
 import lvls from './levels'
 const levels = lvls()
 
-const arrayToObject = function (key, array) {
-  var o = {}
-  array.forEach(function (element) {
+const arrayToObject = (key, array) => {
```

Comment:
Not clear what this function does. Can we either add a comment or give it a better name?